listings_app/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for listings_app project.
+
+Generated by 'django-admin startproject' using Django 2.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Here we tell Django where to look for all our files in respect to this file. This will become important once we have CSS, JS and image assets to find.